### ISSUE TITLE
V 0.1.0

### ISSUE THREAD

**Issue by BobbieBarker** [teambition/merge2#5]
Hey why did you take V 0.1.0 off npm?

**Comment by zensh** [comment]
v0.1.0 is first version and named `gulp-merge`, it is a incomplete version.

Then I imrpoved it, it not only a gulp plugin, but also a common stream tool, and I renamed it to `merge2`.

Then updated, updated, ...I have forgot `gulp-merge` existing.

One day some people reported issue to me https://github.com/teambition/merge2/issues/2 , I had not recognized that `gulp-merge` is my work...

Util yesterday, https://github.com/teambition/merge2/issues/4 refer `gulp-merge`, I can't find it in github, I run `npm info gulp-merge`...

`gulp-merge` is a first incomplete version, it should not be used, so I unpublished it.

Should I publish it with name `gulp-merge` again?

**Comment by zensh** [comment]
@varya have re-publish it to npm, but it still is old version.

**Comment by BobbieBarker** [comment]
we'll if that's the case you should add those details about gulp merge to the nom read me and redirect people to use the propper tool.

**Comment by ExplodingCabbage** [comment]
Just got bitten by the yanked dependency. Despite it being republished, URLs that were already in my npm-shrinkwrap.json like https://registry.npmjs.org/gulp-merge/-/gulp-merge-0.1.0.tgz remain 404s.

Now that I've learned about `merge2`, I intend to go on an editing rampage on Stack Overflow converting all answers that use `gulp-merge` to use `merge2` instead. Others should feel free to go ahead and do so first. I wish I could remember where I first learned about this package so that I could make sure to edit it or get it edited.
